openprescribing/media/js/measures.js

```diff
@@ -1,541 +1,161 @@
-});
-L.mapbox.accessToken = 'pk.eyJ1IjoiYW5uYXBvd2VsbHNtaXRoIiwiYSI6ImNzY1VpYkkifQ.LC_IcHpHfOvWOQCuo5t7Hw';
-
-var measures = {
-    el: {
-        mapPanel: 'map-measure'
-    },
-
-    setUp: function() {
-        this.setUpShowPractices();
-        var _this = this;
-
-        _this.NUM_MONTHS_FOR_RANKING = 6;
-        _this.centiles = ['10', '20', '30', '40', '50', '60', '70', '80', '90'];
-        _this.measure = measureData.measure;
-        _this.measureIsCostBased = (measureData.measureIsCostBased) ? (measureData.measureIsCostBased === 'True') : null;
-        _this.measureIsPercentage = (measureData.measureIsPercentage) ? (measureData.measureIsPercentage === 'True') : null;
-        _this.orgType = measureData.orgType;
-        _this.parentOrg = (measureData.parentOrg) ? measureData.parentOrg : null;
-        _this.orgId = measureData.orgId;
-        _this.orgName = measureData.orgName;
-        _this.rollUpBy = (_this.measure) ? 'org_id': 'measure_id';
-
-        // On long pages, only render some graphs initially, to stop the page choking.
-        _this.allGraphsRendered = false;
-        _this.graphsToRenderInitially = 24;
-
-        if ($('#' + _this.el.mapPanel).length) {
-            _this.setUpMap(_this.orgId, _this.orgType);
-        }
-
-        // All pages have a summary, and a series of panels.
-        var summary_template = Handlebars.compile($('#summary-panel').html()),
-            panel_template = Handlebars.compile($("#measure-panel").html());
-
-        // Get the appropriate URLs for the individual panel measures, and the global measures.
-        var urls = _this.getDataUrls(_this.orgId, _this.orgType, _this.measure);
-        $.when(
-            $.ajax(urls.panelMeasuresUrl),
-            $.ajax(urls.globalMeasuresUrl)
-            ).done(function(panelMeasures, globalMeasures) {
-                var panelData = panelMeasures[0].measures,
-                    globalData = globalMeasures[0].measures,
-                    globalSeries = {};
-
-                // If we're just looking at one measure, calculate the Highcharts x and
-                // y values at this point, so we can re-use them.
-                if (_this.rollUpBy === 'org_id') {
-                    // We're only using one measure. Get the global series.
-                    var series = _.findWhere(globalData, { id: _this.measure});
-                    _.each(_this.centiles, function(i) {
-                        globalSeries[i.toString()] = _this.convertDataForHighcharts(series.data,
-                            true, _this.orgType.toLowerCase(), i);
+    var _ = require('underscore');
+    var mu = require('./src/measure_utils');
+    var chartOptions = require('./src/highcharts-options');
+    require('mapbox.js');
+    var Handlebars = require('handlebars');
+    Highcharts.setOptions({
+        global: { useUTC: false }
+    });
+    L.mapbox.accessToken = 'pk.eyJ1IjoiYW5uYXBvd2VsbHNtaXRoIiwiYSI6ImNzY1VpYkkifQ.LC_IcHpHfOvWOQCuo5t7Hw';
```

Comment:
I'm not sure there's much we can do about it: https://www.mapbox.com/help/rotate-access-token/